R/pkg/R/DataFrame.R

```diff
@@ -557,7 +557,7 @@ setMethod("insertInto",
             jmode <- convertToJSaveMode(ifelse(overwrite, "overwrite", "append"))
             write <- callJMethod(x@sdf, "write")
             write <- callJMethod(write, "mode", jmode)
-            callJMethod(write, "insertInto", tableName)
+            invisible(callJMethod(write, "insertInto", tableName))
```

Comment:
It's a R usability thing: for method not returning any value (in these cases the Scala method is defined as `foo : Unit`), it will print NULL.

so we say `invisible` to get it to not print out the value, that's all